spec/jobs/google/search_keyword_job_spec.rb

```diff
@@ -2,139 +2,277 @@
 
-        described_class.perform_now keyword.id
+          described_class.perform_now keyword.id
 
-        expect(keyword.reload.total_link_count).to eq(14)
-      end
+          expect(keyword.reload.total_link_count).to eq(14)
+        end
 
-      it 'saves returned html as html' do
-        html = file_fixture('html/valid_google.html').read
-        stub_request(:get, %r{google.com/search}).to_return(body: html)
-        keyword = Fabricate(:keyword)
+        it 'saves returned html as html' do
+          html = file_fixture('html/valid_google.html').read
+          stub_request(:get, %r{google.com/search}).to_return(body: html)
+          keyword = Fabricate(:keyword)
 
-        described_class.perform_now keyword.id
+          described_class.perform_now keyword.id
 
-        expect(keyword.reload.html).to eq(html)
-      end
+          expect(keyword.reload.html).to eq(html)
+        end
 
-      it 'saves keyword status as parsed' do
-        stub_request(:get, %r{google.com/search}).to_return(body: file_fixture('html/valid_google.html').read)
-        keyword = Fabricate(:keyword)
+        it 'saves keyword status as parsed' do
+          stub_request(:get, %r{google.com/search}).to_return(body: file_fixture('html/valid_google.html').read)
+          keyword = Fabricate(:keyword)
 
-        described_class.perform_now keyword.id
+          described_class.perform_now keyword.id
 
-        expect(keyword.reload.status).to eq('parsed')
+          expect(keyword.reload.status).to eq('parsed')
+        end
       end
-    end
 
-    context 'given a 422 too many requests error' do
-      it 'does not set any result' do
-        stub_request(:get, %r{google.com/search}).to_return(status: 422)
-        keyword = Fabricate(:keyword)
-
-        described_class.perform_now keyword.id
-        keyword.reload
-
-        expect(
-          [
-            keyword.ads_top_urls,
-            keyword.result_urls,
-            keyword.html,
-            keyword.source
-          ]
-        ).to all(be_nil)
-        expect(
-          [
-            keyword.ads_top_count,
-            keyword.ads_page_count,
-            keyword.result_count,
-            keyword.total_link_count
-          ]
-        ).to all(eq(0))
+      context 'given a 422 too many requests error' do
+        # rubocop:disable RSpec/ExampleLength
+        it 'does not set any result' do
+          stub_request(:get, %r{google.com/search}).to_return(status: 422)
+          keyword = Fabricate(:keyword)
+
+          described_class.perform_now keyword.id
+          keyword.reload
+
+          expect(
+            [
+              keyword.ads_top_urls,
+              keyword.result_urls,
+              keyword.html
+            ]
+          ).to all(be_nil)
+          expect(
+            [
+              keyword.ads_top_count,
+              keyword.ads_page_count,
+              keyword.result_count,
+              keyword.total_link_count
+            ]
+          ).to all(eq(0))
```

Comment:
Please extract the expectation like this, it could improve the readability 🖌️ 

```suggestion
          keyword_urls = [ keyword.ads_top_urls, keyword.result_urls, keyword.html ]
          expect(keyword_urls).to all(be_nil)
```